lib/responses.js

```diff
@@ -312,7 +312,7 @@ function unauthorized(res) {
  * @param {object} res
  */
 function clientError(err, res) {
-  log('error', err.message, { stack: err.stack });
+  log('error', err.message, { stack: err.stack, url: res.locals.url });
```

Comment:
**TODO**: Is a `clientError` really worthy of generating an `error` level log on the server side or should this be a warning?